roles/reset/tasks/main.yml

```diff
@@ -149,6 +149,34 @@
   tags:
     - files
     - dns
+    
```

Comment:
@riverzhang These trailing spaces fail yaml validation. Can you remove them?

Reply:
@chadswen  ok.  I remove it.